flang/test/Lower/intrinsic-procedures/get_command_argument.f90

```diff
@@ -0,0 +1,115 @@
+! RUN: bbc -emit-fir %s -o - | FileCheck --check-prefixes=CHECK,CHECK-32 -DDEFAULT_INTEGER_SIZE=32 %s
+! RUN: flang-new -fc1 -fdefault-integer-8 -emit-fir %s -o - | FileCheck --check-prefixes=CHECK,CHECK-64 -DDEFAULT_INTEGER_SIZE=64 %s
+
+! CHECK-LABEL: func @_QPget_command_argument_test1(
+! CHECK-32-SAME: %[[num:.*]]: !fir.ref<i32>) {
+! CHECK-64-SAME: %[[num:.*]]: !fir.ref<i64>) {
+subroutine get_command_argument_test1(num)
+    integer :: num
+    call get_command_argument(num)
+! CHECK-NOT: fir.call @_FortranAArgumentValue
+! CHECK-NOT: fir.call @_FortranAArgumentLength
+! CHECK-NEXT: return
+end subroutine get_command_argument_test1
+
+! CHECK-LABEL: func @_QPget_command_argument_test2(
+! CHECK-SAME: %[[num:.*]]: !fir.ref<i[[DEFAULT_INTEGER_SIZE]]>,
+! CHECK-SAME: %[[arg:.*]]: !fir.boxchar<1>) {
+subroutine get_command_argument_test2(num, arg)
```

Comment:
[nit] `arg` -> `value` (to match the spec)

Reply:
Done.